src/js/reducers/documentsReducer.js

```diff
@@ -0,0 +1,34 @@
+import * as types from '../actions/action-types';
+
+const initialState = {
+  documents: [],
+};
+
+function documentReducer(state = initialState, action) {
+  switch (action.type) {
+    case types.LOAD_DOCUMENTS_SUCCESS:
```

Comment:
As I said before it would be more clear if you would call this action `LOAD_DOCUMENTS_LIST`, because you will not load whole documents - only their metadata.